workspaces/local-cli/src/commands/status.ts

```diff
@@ -138,49 +151,62 @@ export default class Status extends Command {
       variables: {},
     });
 
-    const endpoints: Endpoint[] = [];
-    const requestBodies: IRequestBodyForTrailParser[] = [];
-    const responseBodies: IResponseBodyForTrailParser[] = [];
-
-    for (const request of requests.data.requests) {
-      endpoints.push({
+    const endpoints = requests.map((request: any) => {
```

Comment:
This is copy pasted from the ui redux store. there's probably a place we can put these in the same area - but it feels like this would go into spectacle. cli-shared is used by the UI and local-cli which is odd but maybe that's the intended location? I'm a little unsure where the best place for these shared data processing / items should live... Perhaps cli-shared is the best place